src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -83,8 +83,11 @@ export const ItemNav = <D,>({ info, config, additionalData,
     useDocEvent("mouseup",    () => setCurPressedObj(undefined));
     useDocEvent("mouseleave", () => setCurPressedObj(undefined));
     function handleItemNavDrag() { // Issue #478
-        if (!currentlyPressedObj || !(currentlyPressedObj instanceof Component))
+        if (!currentlyPressedObj || !(currentlyPressedObj instanceof Component)) {
+            // Disable selection while placing object - resolves #863
+            info.selections.setDisabled(true);
             return;
+        }
```

Comment:
Is this necessary if you're just deselecting everything in the SelectionPopup? I may have forgotten how our conversation went -- i remember deciding on the shared state for `isPlacing` thing but what else did we decide the exact behavior should be?